docs/api-docs/getting-started/api-rate-limits.mdx

```diff
@@ -0,0 +1,120 @@
+# API Rate limits
+
+BigCommerce aims to provide outstanding platform reliability. To ensure dependability, we rely on rate limits for our REST APIs. 
```

Comment:
```suggestion
BigCommerce aims to provide outstanding platform reliability. To ensure reliability, we implement safeguards in form of rate limits for our REST APIs. 
```

Just nitpicking, but to me "rely" is probably too strong word, like it isn't something we rely on for dependability solely.